api/src/main/java/com/capitalone/dashboard/auth/DefaultAuthenticationResponseService.java

```diff
@@ -38,13 +42,35 @@ public void handle(HttpServletResponse response, Authentication authentication)
 			displayName = ((CustomUserDetails) authentication.getPrincipal()).getDisplayName();
 			emailAddress = ((CustomUserDetails) authentication.getPrincipal()).getEmailAddress();
 		}
-		Collection<? extends GrantedAuthority> authorities =
-				userInfoService.getAuthorities(authentication.getName(), firstName, middleName, lastName, displayName, emailAddress, (AuthType)authentication.getDetails());
-		UsernamePasswordAuthenticationToken authenticationWithAuthorities = new UsernamePasswordAuthenticationToken(authentication.getPrincipal(), authentication.getCredentials(), authorities);
-		authenticationWithAuthorities.setDetails(authentication.getDetails());
+
+        UsernamePasswordAuthenticationToken authenticationWithAuthorities = null;
```

Comment:
This class is responsible for adding a JWT token to your response after logging in successfully, you probably don't want to use this, or modify it for the API token.